openshift/kustomize/cron/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine:3.17.1
```

Comment:
Needed a base image that has `bash` installed.  Would have loved an image I didn't have to customize, but this does the trick.